Gruntfile.js

```diff
@@ -55,6 +55,7 @@ const { resolve } = path
 const {
   CI = '',
   LIGHTWEIGHT_CLIENT = 'true',
+  GI_VERSION = `v0.0.1@${new Date().toISOString()}`,
```

Comment:
Version should be fetched from `package.json`.

Reply:
In that case it would currently not change between builds.

Reply:
But I agree that the semver part of the version should come from the` package.json` file. Only the build-specific suffix should be dynamically assigned, assuming we can't fetch it from somewhere for now.